include/deal.II/fe/mapping_p_generic.h

```diff
@@ -421,6 +423,11 @@ class MappingPGeneric : public Mapping<dim, spacedim>
                GeometryInfo<dim>::faces_per_cell *(dim - 1)>
       unit_tangentials;
 
+    /**
+     * TODO
+     */
+    const FiniteElement<dim, spacedim> fe;
```

Comment:
should probably be a smart pointer (this isn't a big deal at this stage)

Reply:
I was thinking if it wasn't better to clone it and put it into a unique pointer!?